CONTRIBUTING.md

```diff
@@ -0,0 +1,36 @@
+# Contributing
+
+## Conduct
+
+All contributors are expected to adhere to the project's [Code of Conduct](CODE_OF_CONDUCT.md).
+
+## Development
+
+To begin development on the VSCode extension for Swift you will need to install [Node.js](https://nodejs.org). On Linux, make sure to install Node.js from its official website or from [NodeSource](https://github.com/nodesource/distributions/) as the version included with your Linux distribution may be outdated.
+
+Next, clone this repository, and in the project directory run `npm install` to install all the dependencies.
+
+When you first open the project in VSCode you will be recommended to also install `ESLint` and `Prettier - Code formatter`. Please do so. We use these to ensure a consistent style and expect everyone who contributes to follow this style as well. 
```

Comment:
There's a third one now: `connor4312.esbuild-problem-matchers`
It provides a problem matcher for ESBuild. Without it, you'll get an error when building.

Reply:
Maybe we should just say "a number of Typescript extensions" to cover our bases and for any future additions/removals

Reply:
@svanimpe i don't have esbuild-problem-matchers installed and it seems to work ok. It was never suggested to me I should install it.